docs/F-Refinements/character-strings.md

```diff
@@ -383,14 +386,14 @@ The `scanf` function will stop accepting input after the character `n` and store
     <td>7</td>
     <td>8</td>
     <td>9</td>
-    <td>10</td>
+    <td><i>10</i><td>
     <td>11</td>
     <td>12</td>
     <td>13</td>
     <td>14</td>
     <td>15</td>
     <td>16</td>
-    <td><b>17</b></td>
+    <td>17</td>
```

Comment:
`<strong>`

Reply:
Will send a PR to fix it in a couple of minute